src/main/java/Duke.java

```diff
@@ -1,10 +1,167 @@
+                userInputSentence.substring(0, Commands.ADD_DEADLINE_COMMAND.length()).equals(Commands.ADD_DEADLINE_COMMAND);
+        boolean isAddEvent = userInputSentence.length() >= Commands.ADD_EVENT_COMMAND.length() &&
+                userInputSentence.substring(0, Commands.ADD_EVENT_COMMAND.length()).equals(Commands.ADD_EVENT_COMMAND);
+
+        TaskType taskTypeToAdd = TaskType.TASK;
+        if (isAddTodo) {
+            taskTypeToAdd = TaskType.TODO;
+        } else if (isAddDeadline) {
+            taskTypeToAdd = TaskType.DEADLINE;
+        } else if (isAddEvent) {
+            taskTypeToAdd = TaskType.EVENT;
+        }
+
+        switch (taskTypeToAdd) {
+        case TODO: {
+            String description = userInputSentence.substring(Commands.ADD_TODO_COMMAND.length());
+            tasks[inputStoredIndex] = new Todo(description);
+            printSuccessfulAddedMessage();
+            inputStoredIndex++;
+            break;
+        }
+        case DEADLINE: {
+            String[] splitCommands = userInputSentence.substring(Commands.ADD_DEADLINE_COMMAND.length()).split("/by ");
+            tasks[inputStoredIndex] = new Deadline(splitCommands[0], splitCommands[1]);
+            printSuccessfulAddedMessage();
+            inputStoredIndex++;
+            break;
+        }
+        case EVENT: {
+            String[] splitCommands = userInputSentence.substring(Commands.ADD_EVENT_COMMAND.length()).split("/at ");
+            tasks[inputStoredIndex] = new Event(splitCommands[0],splitCommands[1]);
+            printSuccessfulAddedMessage();
+            inputStoredIndex++;
+            break;
+        }
+        default: {
+            /* If the user doesn't specify the task type to add, the reminder will be given. */
+            System.out.println(SEPARATOR);
+            System.out.println("Please specify the task type.");
+            System.out.println(SEPARATOR);
+        }
+        }
+    }
+
+    private static void setTaskDone(Integer setTaskId) {
+        try {
+            Task targetTask = tasks[setTaskId];
+            targetTask.setDone();
+
+            System.out.println(SEPARATOR);
+            System.out.printf("Nice! I've marked this task as done: \n[%s] %s\n",
+                    targetTask.getStatusIcon(),targetTask.description);
+            System.out.println(SEPARATOR);
+        }
+        catch (NullPointerException e) {
+            System.out.printf("Task %d does not exist.\n", setTaskId);
+        }
+    }
+
+    private static void printSuccessfulAddedMessage() {
+        System.out.println(SEPARATOR);
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(tasks[inputStoredIndex]);
+        System.out.printf("Now you have %d tasks in the list.\n",Task.getNumTotalTasks());
+        System.out.println(SEPARATOR);
+    }
+
+    public static void sayGoodbye(){
+        String goodbyeWords = "Bye. Hope to see you again soon!";
```

Comment:
Consider renaming to goodbyeMessage, as goodbyeWords may sound like it's referring to an array of words